landing-page/content/common/how-to-release.md

```diff
@@ -69,6 +81,14 @@ Set it as remote with name `apache` for release if it is not already set up.
 
 ## Creating a release candidate
 
+### Initiate a discussion about the release with the community
+
+This step can be useful to gather ongoing patches that the community thinks should be in the upcoming release.
+
+The communication can be started via a [DISCUSS] mail on the dev@ channel and the desired tickets can be added to the github milestone of the next release.
+
+Note, creating a milestone in github requires a committer. However, a non-committer can assign tasks to a milestone if added to the list of collaborators in [.asf.yaml](https://github.com/apache/iceberg/blob/master/.asf.yaml)
```

Comment:
We should also mention that the release status is discussed during each community sync meeting. Release manager should join the meeting to report status and discuss any release blocker.

Reply:
Sure, added